charts/s3gw/templates/ingress-traefik.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     traefik.ingress.kubernetes.io/router.tls: "true"
     traefik.ingress.kubernetes.io/router.middlewares:
       '{{ .Release.Namespace }}-{{ include "s3gw.CORSMiddlewareName" . }}@kubernetescrd'
-    cert-manager.io/cluster-issuer: {{ default .Values.tlsIssuer .Values.customTlsIssuer | quote }}
```

Comment:
This means that we don't support specifying customized tls issuer at all. I think we should allow the user to specify their own tls issuer

Reply:
I agree with @PhanLe1010, it should be possible for the user to specify its tls issuer.

Reply:
I fixed it. I was not aware this was a feature.

Reply:
I dunno if we originally designed that to be an official feature, but we would break the existing if we changed that.